src/main/java/duke/utility/Parser.java

```diff
@@ -0,0 +1,48 @@
+package duke.utility;
+
+import duke.command.*;
```

Comment:
Just for the sake for stupid coding standard, listing all the command explicitly might be better.
[https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements](url)